### ISSUE TITLE
CT Armor Modifier Stuck at Zero

### ISSUE THREAD

**Issue by Not-Stabby** [xdy/twodsix-foundryvtt#1557]
When using the CT ruleset, the "Armor Modifier" field defaults to a value of 0, and forcibly reverts to 0 no matter what you enter. This makes it seemingly impossible to implement armors introduced in the supplements such as the Flak Jacket (Which is treated as Cloth+1, as in Cloth but attacks against it are made with a +1) and others.

I am unsure if this is a bug or if I've simply misinterpreted the function of the Armor Modifier and this is its intended behavior, so I am unsure whether to label this a request or not. Either way, the ability to add new armor items to CT would be greatly appreciated.

(Apologies for posting so many of these by the way.)

**Comment by marvin9257** [comment]
Yup...its a bug.  Somehow the ArmorDM changes got overwritten.  Thanks!  And please keep finding bugs.  It is incredibly helpful.

**Comment by Not-Stabby** [comment]
> Yup...it's a bug. Somehow the ArmorDM changes got overwritten. Thanks! And please keep finding bugs. It is incredibly helpful.

Will do!
